overrides/php/default.nix

```diff
@@ -0,0 +1,147 @@
+{
+  lib,
+  pkgs,
+  ...
+}: let
+  l = lib // builtins;
+in {
+  "phpstan/phpstan-phpunit" = {
+    fix-autoload-dev = {
```

Comment:
It seems like this override is required for many packages. Could this logic just be integrated into the builder? There are probably many more packages that will require it and we would otherwise have to add an override for each of them.

Reply:
Yes, actually now I realize that we don't want to apply these `dev` settings for dependencies at all. 
So just adding `--no-dev` flag for dependencies to `composer install` will take care of this.